test/test_files/copy/copy_long_string.test

```diff
@@ -0,0 +1,19 @@
+-GROUP CopyLongStringTest
+-DATASET CSV copy-fault-tests/long-string
+
+--
+
+-CASE LongStringError
+-DEFINE AliceStr REPEAT 819 "Alice"
```

Comment:
I assume you're following Postgres's `repeat` function? If so, let's switch the order of these two params. `REPEAT "Alice" 819` is more consistent with existing functions.
If not, I suggest the principle that we always follow existing syntaxes, instead of coin new ones.